roles/contiv_network/templates/aci_gw.j2

```diff
@@ -19,7 +19,7 @@ start)
     -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
     -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
     --name=contiv-aci-gw \
-    contiv/aci-gw
+    {{ aci_gw_version }}
```

Comment:
> If someone wants to use his own custom image then just tag wont be sufficient.

hmm, I am not sure if I understand. You mean someone ca use a non aci-gw image built by them?

> Default version is mentioned in net_demo_installer script.

what I meant was ansible shall have a default defined too, as this will be use by other projects as well. Please se how it is done for other plays. For instance checkout etcd container download task: https://github.com/contiv/ansible/blob/master/roles/etcd/tasks/main.yml#L20